src/driver.c

```diff
@@ -653,8 +661,7 @@ partial (int tbl, int s)
  * even built on non-Linux platforms.
  */
 
-#if ( defined(_POSIX_C_SOURCE) || !defined(WIN32) )
-
+#ifdef CAN_PLOAD
```

Comment:
if you're introducing a new preprocessor macro, please use less abbreviation, e.g. `CAN_PARALLELIZE_DATA_GENERATION_LOAD` or `CAN_PARALLELIZE_DATA_GENERATION` or something.

Reply:
Ok